docs/source/tutorials/experimentation/exp_plotting.rst

```diff
@@ -0,0 +1,149 @@
+     num_at_level27: [36, 35, 25, 28, 45, 37, 27, 36, 46, 27, 37, 26, 37, 58, 31, 36, 26, 28, 33, 39]
+     num_at_level81: [20, 13, 12, 11, 23, 20, 13, 20, 23, 10, 13, 9, 18, 31, 16, 18, 11, 16, 19, 21]
+   [ASHA-RELU]:
+     num_at_level1: [599, 670, 682, 817, 608, 585, 770, 397, 613, 721, 599, 601, 618, 718, 613, 674, 715, 638, 598, 652]
+     num_at_level3: [201, 246, 242, 277, 225, 209, 282, 140, 212, 245, 202, 205, 215, 245, 207, 239, 238, 224, 221, 234]
+     num_at_level9: [75, 94, 94, 100, 89, 92, 101, 60, 78, 89, 76, 82, 80, 98, 86, 96, 83, 84, 90, 91]
+     num_at_level27: [37, 43, 36, 34, 40, 45, 39, 35, 34, 31, 40, 40, 38, 39, 35, 34, 29, 34, 41, 35]
+     num_at_level81: [23, 19, 14, 13, 19, 21, 15, 24, 17, 13, 20, 18, 19, 18, 20, 16, 13, 15, 22, 17]
+   [MOBSTER-RELU]:
+     num_at_level1: [241, 319, 352, 438, 354, 386, 197, 262, 203, 387, 320, 139, 359, 401, 334, 294, 361, 403, 178, 141]
+     num_at_level3: [110, 156, 135, 166, 138, 143, 104, 124, 95, 136, 133, 71, 133, 151, 130, 122, 134, 151, 92, 74]
+     num_at_level9: [50, 83, 59, 75, 59, 55, 57, 72, 53, 53, 58, 40, 62, 63, 61, 54, 52, 65, 48, 47]
+     num_at_level27: [31, 51, 29, 31, 29, 23, 39, 38, 36, 20, 29, 36, 32, 29, 32, 29, 24, 27, 31, 34]
+     num_at_level81: [20, 35, 12, 11, 12, 15, 22, 18, 26, 12, 16, 27, 16, 15, 20, 15, 15, 13, 18, 22]
+
+* There are four subfigures arranged as two-by-two matrix. Each contains two
+  curves in bold, along with confidence intervals. The horizontal axis
+  depicts wall-clock time, and on the vertical axis, we show the best
+  metric value found until this time.
+* More general, the data from our 640 experiments can be grouped w.r.t.
+  subplot, then *setup*. Each setup gives rise to one curve (bold, with
+  confidence band). Subplots are optional, the default is to plot a single
+  figure.
+* The function ``metadata_to_setup`` maps the metadata stored for an experiment
+  to the setup name. In our basic case, the setup is simply the name of the
+  method.
+* The function ``metadata_to_subplot`` maps the metadata to the subplot
+  index (0, 1, 2, 3). We group setups with the same configuration space,
+  but also split multi-fidelity and single-fidelity methods.
+* Once the data is grouped w.r.t. benchmark, then subplot (optional), then
+  setup, we should be left with 20 experiments, one for each seed. These 20
+  curves are now interpolated to a common grid, and at each grid point, the
+  20 values are aggregated into ``lower``, ``aggregate``, ``upper``. In the
+  figure, ``aggregate`` is shown in bold, and ``lower``, ``upper`` in dashed.
+  Different aggregation modes are supported (selected by
+  ``plot_params.aggregate_mode``).
+* We pass ``extra_results_keys`` to the
+  :meth:`~syne_tune.experiments.ComparativeResults.plot` method in order to
+  also retrieve extra results. This method returns a dictionary, whose
+  "extra_results" entry is what we need.
+
+Advanced Experimenteering
```

Comment:
Is the spelling of `Experimenteering` intentional?

Reply:
I change it